src/local-storage-assignment-cache.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { LocalStorageAssignmentCache } from './local-storage-assignment-cache';
```

Comment:
this was in `index.spec.ts` but I split it out into its own file

Reply:
nice thinking 🧹